src/components/templates/PartyMap/PartyMap.scss

```diff
@@ -1,5 +1,27 @@
 @import "scss/constants.scss";
 
+// TODO: do we need any of this any more?
```

Comment:
No, you can delete them.

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/840/commits/7f23bce63ad968079ab80e5d68af013fb7fc2eab